### DISCUSSION TITLE
Path problem after last update

### DISCUSSION THREAD

**Discussion by OmarAlhaji** [FilledStacks/stacked#1070] [Q&A]
I had a problem. I was using the CLI and it was working very normally through the CMD. I received a message that there was a new update for stacked. I did a stacked update and then I had this problem and I was not able to solve it.

C:\Users\Omar>stacked create app search
Unhandled exception:
Bad state: The "HOME" environment variable is not set. This package (and POSIX) requires that HOME be set.
#0      _getDirectory (package:xdg_directories/xdg_directories.dart:120:5)
#1      _directoryFromEnvironmentWithFallback (package:xdg_directories/xdg_directories.dart:109:12)
#2      configHome (package:xdg_directories/xdg_directories.dart:152:5)
#3      PathService.configHome (package:stacked_cli/src/services/path_service.dart:41:35)
#4      PosthogService.init (package:stacked_cli/src/services/posthog_service.dart:42:31)
#5      setupLocator (package:stacked_cli/src/locator.dart:27:24)
#6      main (file:///C:/Users/Omar/AppData/Local/Pub/Cache/hosted/pub.dev/stacked_cli-1.12.6/bin/stacked.dart:18:9)
#7      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#8      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

Please help if you have a solution

**Comment by ferrarafer** [comment]
Hi @OmarAlhaji !

1. Which is your stacked cli version ?
2. Where do you store your `stacked.json` config file ?
3. Can you check if [this](https://github.com/Stacked-Org/stacked/issues/1063#issuecomment-1837534278) works for you?
